docs/versioning.md

```diff
@@ -2,33 +2,22 @@
 
 ## Introduction
 
-Fedora 4 provides support for making snapshots of the current state of a resource which in turn become part of the resources version history.  The RDF for historic version snapshots may be browsed and old non-RDF content may be downloaded.  Furthermore, an object or subgraph may be reverted to the state that it existed in a historic version.
-
-The Fedora API Specification supports all the versioning capabilities for Fedora 4, the api for accessing these features will completely change.  To wit, the Fedora API specification conforms to the [Memento](https://mementoweb.org/guide/rfc/) specification.  Memento brings with capabilities that were not available in Fedora 4.  Below we will explore the most vital of these features as well as compare Fedora 4 with the Memento way of doing versioning.
+The Fedora API specification conforms to the [Memento](https://mementoweb.org/guide/rfc/) specification.  Memento brings with it all the versioning capabilities of Fedora 4 along with some new features. Below we will explore the most vital of these new features as well as compare the Fedora 4 with the Memento way of doing versioning.
```

Comment:
> Memento brings with it all the versioning capabilities of Fedora 4 along with some new features.

Maybe this is backwards? It could be "Fedora 4 brings with it all the versioning capabilities of Memento along with some new features."